src/ussd_optout_rapidpro.js

```diff
@@ -1,29 +1,333 @@
+                        return "state_loss_optout";
+                    } else {
+                        return "state_delete_research_info";
+                      }  
+                }
+            });
+        });
+
+        self.add("state_loss_optout", function(name) {
+            return new ChoiceState(name, {
+                question: $("We're sorry for your loss. Would you like to receive a small set of " +
+                            "MomConnect messages that could help you during this difficult time?"),
+                error: $(
+                    "Sorry, please reply with the number next to your answer. " +
+                    "Would you like to receive a small set of MomConnect messages?"
+                ),
+                choices: [
+                    new Choice("yes", $("Yes")),
+                    new Choice("no", $("No"))
+                ],
+                next: function(choice) {
+                    if(choice.value === "yes") {
+                        return "state_delete_info_for_loss";
+                    } else {
+                        return "state_delete_research_info";
+                    }
+                }
+            });
+        });
+
+        self.add("state_delete_info_for_loss", function(name) {
+            return new ChoiceState(name, {
+                question: $("You'll get support msgs. We hold your info for historical/research/statistical " +
+                            "reasons. Do you want us to delete it after you stop getting msgs?"),
+                error: $(
+                    "Sorry, please reply with the number next to your answer. " +
+                    "Do you want us to delete it after you stop getting msgs?"
+                ),
+                choices: [
+                    new Choice("yes", $("Yes")),
+                    new Choice("no", $("No"))
                 ],
-                next: "state_start"
+                next: "state_trigger_rapidpro_flow"
+            });
+        });
+
+        self.add("state_delete_research_info", function(name) {
+            return new ChoiceState(name, {
+                question: $("We hold your info for historical/research/statistical reasons after " +
+                            "you opt out. Do you want to delete your info after you stop getting messages?"),
+                error: $(
+                    "Sorry, please reply with the number next to your answer. " +
+                    "Do you want to delete your info after you stop getting messages?"
+                ),
+                choices: [
+                    new Choice("yes", $("Yes")),
+                    new Choice("no", $("No"))
+                ],
+                next: function(choice) {
+                    if(choice.value === "yes") {
+                        return "state_info_deleted";
+                    } else {
+                        return self.states.create("state_trigger_rapidpro_flow");
```

Comment:
We can just return the string `"state_trigger_rapidpro_flow"` here, like we do for `state_info_deleted`

Reply:
okay, thanks